src/test/scala/group/scala/karazin/circe/literal/extras/arbitraries/ArbitrariesJavaDateTime.scala

```diff
@@ -48,18 +48,18 @@ trait ArbitrariesJavaDateTime {
   val yearMonthGenerator: Gen[YearMonth] =
     for {
       y <- Gen.choose(Year.MIN_VALUE, Year.MAX_VALUE)
-      m <- Gen.choose(1,12)
+      m <- Gen.choose(1, 12)
     } yield YearMonth.of(y, m)
 
   val zonedDateTimeGenerator: Gen[ZonedDateTime] =
     for {
-      ld <- Gen.choose(LocalDate.MIN, LocalDate.MAX)
-      lt <- Gen.choose(LocalTime.MIN, LocalTime.MAX)
-      z  <- Gen.oneOf("EST", "HST", "MST", "ACT", "AET", "AGT",
+      localDate <- Gen.choose(LocalDate.MIN, LocalDate.MAX)
+      localTime <- Gen.choose(LocalTime.MIN, LocalTime.MAX)
+      zone  <- Gen.oneOf("EST", "HST", "MST", "ACT", "AET", "AGT",
```

Comment:
There's a small issue here - ZoneId.of(...) requires an actual offset name (e.g., Asia/Aden), otherwise you'll get ZoneRulesException. I think you could use ZoneId.SHORT_IDS.values() or ZoneId.getAvailableZoneIds (for the complete list) in your `Gen.oneOf()` and it should do the job.